todoComments/todoComments.md

```diff
@@ -0,0 +1,185 @@
+| 30 | ISSUE when many meters selected they are cut off. | src/client/app/components/ThreeDPillComponent.tsx | Needs Issue Created | - | Pill Components go below the container and out of view when many meters are selected and the width of the screen is below a certain point. Allowing vertical scroll here could be a potential fix. |
+| 31 | Provide default link when there are issues fetching version number | src/client/app/components/TooltipHelpComponent.tsx | Needs Issue Created | - | If there is a problem with fetching the OED version, Tooltip links will display "..." in place of the <a> tag link. A default link could be used instead of the "..." |
+| 32 | This is trying to fix the fact that when the rate menu appears and disappears and you don't go to a new page then the help pop up does not occur. This works reasonably well but you have to click the help icon twice to see the pop up in this case. A better fix would be desirable in the long term since there have been issues with tooltips in multiple places. | src/client/app/components/TooltipMarkerComponent.tsx | Related To An Issue | [Github issue #988](https://github.com/OpenEnergyDashboard/OED/issues/988)  | Issue 988 react-tooltip upgrade from v4 to v5 may solve this problem |
+| 33 | migrate ReactRouter v6 & hooks | src/client/app/components/UnsavedWarningComponent.tsx | Unsure | - | It looks like react-router has already been upgraded to v6. Unsure if there are specific hooks that were in mind that still need to be implemented but this seems okay. React-router does suggest not blocking users requests and to try to use a different method |
+| 34 | translate me | src/client/app/components/UnsavedWarningComponent.tsx | No Issue Needed | - | Looks like this translation has already been added. I believe this TODO comment can be removed |
+| 35 | translate me | src/client/app/components/UnsavedWarningComponent.tsx | No Issue Needed | - | Looks like this translation has already been added. I believe this TODO comment can be removed |
+| 36 | Add warning for invalid data | src/client/app/components/admin/PreferencesComponent.tsx | Needs Issue Created | - | If preferencesApi.useGetPreferencesQuery() has a problem fetching data, an error may be thrown while trying to set preferences. |
+| 37 | looks kind of bad make a better visible notification | src/client/app/components/conversion/CreateConversionModalComponent.tsx | Needs Issue Created | - | The error reason message in the footer of the modal needs an update to how it looks while showing the reason of the conversion error |
+| 38 | This feature is not working perfectly so disabling from web page but allowing in curl. Rest of changes left so easy to add back in. | src/client/app/components/csv/ReadingsCSVUploadComponent.tsx | Unsure | - | Checkbox for useMeterZone is commented out for now. Some investigating needs to be done to see why it causes problems on the web |
```

Comment:
Actually, this feature was added to help a site import data. It works but can have issues. Thus, it is only allowed for script uploading since that was the need. I started to add it to the web page input of import but decided to not allow it at this time. Thus, the code was commented out.

Reply:
I can add this new info to the comment note. Should I mark this as needing an issue created in the future or do you think a different status would be more appropriate?

Reply:
Yes, you can update the note. I don't think we need an issue yet as there is no plan to make this generally available due to its limitations.